src/utils.ts

```diff
@@ -710,6 +710,12 @@ export function processTypes(
     addIdentiferToImports(identifier);
   }
 
+  function getServiceName(name: string): string {
+    return name.toLowerCase().endsWith("service")
+      ? name.replace(new RegExp("(?:s)*service", "i"), "")
+      : name;
+  }
+
```

Comment:
I think `google-protobuf` always appends `Service` -- did you check to see if that is the case?

If that is true, this can simply be inlined as `name.replace('Service', '')`.

Reply:
Ah I'm wrong about this -- I _do_ add the service suffix. Instead of the above, we'll want to drop `Service` from here: https://github.com/tatethurston/TwirpScript/blob/main/src/autogenerate/index.ts#L421-L426

Reply:
> Ah I'm wrong about this -- I _do_ add the service suffix. Instead of the above, we'll want to drop `Service` from here: https://github.com/tatethurston/TwirpScript/blob/main/src/autogenerate/index.ts#L421-L426

Yeah I spotted this and originally I had this inlined in a few places in that file but it seemed silly to repeatedly run the regex. Eventually I figured that setting the name in a way so that we can assume safely that it doesn't have `Service` inside it would be best. 

Open to more thoughts on this though - I'm not as familiar with the codebase as you are and didn't spend too long playing around. Was mostly trying to validate this was a sane idea.

Reply:
Let's delete the two references to` Service` from autogenerate so the service name ends up exactly as written in the generated code.